spec/index.html

```diff
@@ -614,38 +622,38 @@ <h3>Examples</h3>
           <caption>Hash to blank nodes map for unique hashes</caption>
           <thead><tr>
             <th><var>hash</var></th>
-            <th><var>label(s)</var></th>
+            <th><var>blank node(s)</var></th>
           </tr></thead>
           <tbody>
             <tr>
               <td>`21d1dd5ba21f3dee9d76c0c00c260fa6f5d5d65315099e553026f4828d0dc77a`</td>
-              <td>`_:e0`</td>
+              <td>`e0`</td>
             </tr>
             <tr>
               <td>`6fa0b9bdb376852b5743ff39ca4cbf7ea14d34966b2828478fbf222e7c764473`</td>
-              <td>`_:e1`</td>
+              <td>`e1`</td>
             </tr>
           </tbody>
         </table>
 
         <p><a href="#ca-4">Step 4</a> creates canonical identifiers for each
           blank node which has a unique hash:</p>
 
-        <table id="ex-ca-unique-1d-labels"
+        <table id="ex-ca-unique-1d-identifiers"
                class="hash-related numbered">
-          <caption>Canonical labels for unique hashes</caption>
+          <caption>Canonical identifiers for blank nodes with unique hashes</caption>
           <thead><tr>
-            <th><var>label</var></th>
+            <th><var>blank node</var></th>
```

Comment:
Here, I think the use of "blank node" vs "label" is justified, as the introducing paragraph uses the term "blank node", so it should not be confusing.